### ISSUE TITLE
Feature - 

### ISSUE THREAD

**Issue by kazisobaeid1** [Gabsii/spoti-vote#274]
## TLDR; ##

* add a short description here about your PR
* use bullet points to describe what happened
* try to be precise and short in your descriptions

## What to test & how to test ##

* describe the necessary steps here, how your colleagues can test your PR
* also describe what they should look for
* give 1-2 exact examples

## Additional Information ##

If needed, add here some additional information which may be useful for the PR. Otherwise just delete this part.

## Definition of Done ##

- Unit Tests for components are created / updated (npm run test)
- All tests are running successfully
- No linting errors / no console log errors (browser)
- Cross browser testing (Chrome, Firefox, Safari, IE, mobile)
- Code was reviewed by a second developer

**Comment by Gabsii** [comment]
Hi @kazisobaeid1 !

I see that you've opened a feature request for spoti-vote. Could you please also add a description of what you want? 
Otherwise I'll close this issue.
